### ISSUE TITLE
[semantic-issue] Add semantics for sklearn GradientBoostingClassifier

### ISSUE THREAD

**Issue by osryde** [spangea/Pyra#175]
**Description**
Invokes the constructor to instantiate a GradientBoostingClassifier object.
Notebook: 0-91-accuracy-score-for-4-models-lr-rc-gbc-rfc.py

**Minimal running example**
```
from sklearn.ensemble import GradientBoostingClassifier

gb = GradientBoostingClassifier()
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html

**Note (optional)**
Returns an object of GradientBoostingClassifier type.
